index.bs

```diff
@@ -12032,22 +12034,18 @@ defined in section [[#platformobjectset]].
 </div>
 
 
-<h4 id="getownproperty">Platform object \[[GetOwnProperty]] method</h4>
+<h4 id="legacy-platform-object-getownproperty" oldids="getownproperty">Legacy platform object \[[GetOwnProperty]] method</h4>
 
 <div algorithm="to invoke the internal [[GetOwnProperty]] method of platform objects">
 
-    The internal \[[GetOwnProperty]] method of every [=platform object=] |O|
-    that implements an [=interface=] which
-    [=support indexed properties|supports indexed=] or
-    [=support named properties|named properties=]
+    The internal \[[GetOwnProperty]] method of every [=legacy platform object=] |O|
     must behave as follows when called with property name |P|:
 
-    1.  Return the result of invoking the [=The PlatformObjectGetOwnProperty abstract operation|PlatformObjectGetOwnProperty=]
-        abstract operation with |O|, |P|, and <emu-val>false</emu-val> as arguments.
+    1. Return [=LegacyPlatformObjectGetOwnProperty=](|O|, |P|, <emu-val>false</emu-val>).
 </div>
 
 
-<h4 id="invoking-indexed-setter">Invoking a platform object indexed property setter</h4>
+<h4 id="invoking-indexed-setter">Invoking a legacy platform object indexed property setter</h4>
```

Comment:
As a follow-up: IMO this should be an abstract op, like LegacyPlatformObjectGetOwnProperty. And all the abstract ops should move after the method definitions, to the bottom of this section.